src/include/common/types/types.h

```diff
@@ -287,80 +288,113 @@ class LogicalType {
         const std::vector<std::unique_ptr<LogicalType>>& types);
 
     static std::unique_ptr<LogicalType> BOOL() {
-        return std::make_unique<LogicalType>(LogicalTypeID::BOOL);
+        return std::unique_ptr<LogicalType>(new LogicalType(LogicalTypeID::BOOL));
```

Comment:
I thought `make_unique` is preferred compared to `std::unique_ptr<>(new object)`. Is there a reason that you are changing?

Reply:
I'd tried making the constructors private, but only the two-argument one still is. I can revert the change for the simple types, but make_unique doesn't work with private constructors.